src/analyze-trace-file.ts

```diff
@@ -157,10 +160,13 @@ async function main(): Promise<void> {
 
         const parent = stack[i];
         const duration = span.end - span.start;
-        if (duration >= thresholdDuration || duration >= minPercentage * (parent.end - parent.start)) {
+        if (duration >= minDuration && (duration >= thresholdDuration || (span.event?.cat !== "program" && duration >= minPercentage * (parent.end - parent.start)))) {
```

Comment:
I was trying to exclude `findSourceFile` events from being included simply because they were large percentages of their parents.  The parent-child relationship is less important for `findSourceFile` since eliminating the parent just moves the child to the next event that requires it (think about trying to eliminate a `@types/node` dependency by cleaning up the first file that pulls it in).